Networking/Networking/Remote/ProductAttributesRemote.swift

```diff
@@ -0,0 +1,121 @@
+import Foundation
+
+/// Product Attributes: Remote Endpoints for fetching global product attributes.
+///
+public final class ProductAttributesRemote: Remote {
+
+    // MARK: - Product Attributes
+
+    /// Retrieves all of the global`ProductAttribute` available.
```

Comment:
I am not sure if this actually more accurate, but maybe "Retrieves all of the `ProductAttribute`s available for a given site" sounds better than global, which might be a bit too broad?

Reply:
Ops sorry for not being totally clear here. There are two types of `ProductAttribute`. Global Attributes and Custom attributes. The global are the `ProductAttribute`s fetched using these endpoints, the custom attributes are the one added inside the specific product variation. I added the word `global` because this `*Remote` class is useful only for fetching global attributes :) more detail here p99K0U-1Mb

Reply:
Oh, I see, thanks for the clarification! 🙇